doc/src/Tools.txt

```diff
@@ -78,7 +78,8 @@ Post-processing tools :h3
 "reax"_#reax_tool,
 "smd"_#smd,
 "spin"_#spin,
-"xmgrace"_#xmgrace :tb(c=6,ea=c,a=l)
+"xmgrace"_#xmgrace,
+"replica"_#replica :tb(c=6,ea=c,a=l)
```

Comment:
This list is supposed to be in alphabetic order.

Reply:
done